pkg/controller/resourcequota/resource_quota_controller.go

```diff
@@ -301,7 +301,7 @@ func (rq *ResourceQuotaController) syncResourceQuotaFromKey(key string) (err err
 	if err != nil {
 		return err
 	}
-	quota, err := rq.rqLister.ResourceQuotas(namespace).Get(name)
+	resourceQuota, err := rq.rqLister.ResourceQuotas(namespace).Get(name)
```

Comment:
```suggestion
	rq, err := rq.rqLister.ResourceQuotas(namespace).Get(name)
```

> Variable names in Go should be short rather than long.  This is especially true for local variables with limited scope.  Prefer `c` to `lineCount`.  Prefer `i` to `sliceIndex`.
>
> The basic rule: the further from its declaration that a name is used, the more descriptive the name must be. For a method receiver, one or two letters is sufficient. Common variables such as loop indices and readers can be a single letter (`i`, `r`). More unusual things and global variables need more descriptive names.

https://github.com/golang/go/wiki/CodeReviewComments#variable-names